pkgs/games/liberal-crime-squad-git/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, stdenv, ncurses, autoconf, automake , SDL2, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  version = "2016-03-03";
+  name = "liberal-crime-squad-git-${version}";
```

Comment:
I was following the manual: http://nixos.org/nixpkgs/manual/#sec-package-naming. It's possible a non-git version could be added, it's just that upstream doesn't seem to have released one in years.